health.go

```diff
@@ -9,6 +9,7 @@ import (
 	"time"
 	"bufio"
 	"os"
+	"github.com/kkdai/maglev"
```

Comment:
dgryski has responded to request and will be uploading an MIT license. I'll edit use dgryski's repo directly without referencing kkdai's implementation, which does look suspiciously copied.

Resolved!